agent/main.go

```diff
@@ -84,36 +87,39 @@ func main() {
 	flag.StringVar(&namespace, "namespace", "default", "Namespace in the management cluster where you would like to register this host")
 	flag.Var(&labels, "label", "labels to attach to the ByoHost CR in the form labelname=labelVal for e.g. '--label site=apac --label cores=2'")
 	flag.StringVar(&metricsbindaddress, "metricsbindaddress", ":8080", "metricsbindaddress is the TCP address that the controller should bind to for serving prometheus metrics.It can be set to \"0\" to disable the metrics serving")
+	flag.StringVar(&downloadpath, "downloadpath", "/var/lib", "File System path to keep the downloads")
```

Comment:
I guess download path is just to store the download files, right? In that way we can just hard-code this folder instead of ask the user to provide this.  Just like other software that have their default install directory.

Reply:
This is opt-in, with a default. It should be /var/lib/**someting**

Reply:
Makes sense. Created a new issue to track this - https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/218